test/proto/service.proto

```diff
@@ -8,29 +8,37 @@ package haskell.grpc.mqtt.test;
 
 service TestService {
   /* Unary Service */
-  rpc NormalCall(TwoInts) returns (OneInt) {}
+  rpc NormalCall(TwoInts) returns (OneInt) {
+  }
 
   /* Server Streaming Service */
-  rpc ServerStreamCall(StreamRequest) returns (stream StreamReply) {}
+  rpc ServerStreamCall(StreamRequest) returns (stream StreamReply) {
+  }
 
   /* Client Streaming Service */
-  rpc ClientStreamCall(stream OneInt) returns (OneInt) {}
+  rpc ClientStreamCall(stream OneInt) returns (OneInt) {
+  }
 
   /* Simple bidi streaming: ping-pong echo */
-  rpc BiDiStreamCall(stream BiDiRequestReply) returns (stream BiDiRequestReply) {}
+  rpc BiDiStreamCall(stream BiDiRequestReply) returns (stream BiDiRequestReply) {
+    option haskell.grpc.mqtt.client_clevel = CLEVEL_10;
+    option haskell.grpc.mqtt.server_clevel = CLEVEL_10;
```

Comment:
Now that I'm seeing it in a proto, is it possible to mix and match CLevels between the client and server? Or do they need to match? If they need to match, do we need distinct server and client clevel options?

Reply:
That's pretty cool.